__tests__/app.test.js

```diff
@@ -0,0 +1,52 @@
+const seed = require("../db/seeds/seed");
+const supertest = require("supertest");
+const data = require("../db/data/test-data/index.js");
+const connection = require("../db/connection.js");
+const app = require("../app.js");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return connection.end();
+});
+
+describe("GET /api/topics", () => {
+  test.only("200: responds with an array of topic objects", () => {
```

Comment:
Don't forget to remove .only before pushing, we don't know if your other tests pass :)